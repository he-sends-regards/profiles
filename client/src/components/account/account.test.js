import React from 'react';
import renderer from 'react-test-renderer';
import Account from './account.jsx';

it('renders correctly', () => {
  const tree = renderer
      .create(<Account />)
      .toJSON();
  expect(tree).toMatchSnapshot();
});
