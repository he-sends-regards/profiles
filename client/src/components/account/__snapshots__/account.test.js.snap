// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="account"
>
  <div
    className="mx-0 row"
  >
    <div
      className="col-sm-2"
    >
      <div
        className="flex-column text-center nav nav-pills"
        onKeyDown={[Function]}
        role="tablist"
      >
        <h3>
          <img
            alt="Account logo"
            src="account-logo.svg"
            style={
              Object {
                "filter": "invert()",
              }
            }
            width="40px"
          />
          <br />
        </h3>
        <br />
        <div
          className="nav-item"
        >
          <a
            aria-controls="left-tabs-example-tabpane-MyProfiles"
            aria-selected={false}
            className="nav-link"
            data-rb-event-key="MyProfiles"
            href="#"
            id="left-tabs-example-tab-MyProfiles"
            onClick={[Function]}
            onKeyDown={[Function]}
            role="tab"
          >
            My profiles
          </a>
        </div>
        <br />
        <button
          className="account__logout-btn btn btn-outline-danger"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Logout
        </button>
      </div>
    </div>
    <div
      className="col-sm-10"
    >
      <div
        className="tab-content"
      >
        <div
          aria-hidden={true}
          aria-labelledby="left-tabs-example-tab-MyProfiles"
          className="fade tab-pane"
          id="left-tabs-example-tabpane-MyProfiles"
          role="tabpanel"
        >
          <div
            className="profiles-list"
          >
            <div
              style={
                Object {
                  "alignItems": "center",
                  "display": "flex",
                  "height": "200px",
                  "justifyContent": "center",
                  "width": "200px",
                }
              }
            >
              <button
                className="btn btn-primary btn-lg"
                disabled={false}
                onClick={[Function]}
                style={
                  Object {
                    "alignItems": "center",
                    "display": "flex",
                    "height": "30%",
                    "justifyContent": "center",
                    "width": "30%",
                  }
                }
                type="button"
              >
                +
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
